sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/ied/InputsAdapter.java

```diff
@@ -94,6 +95,7 @@ protected String elementXPath() {
      * @return list of encountered errors
      */
     public List<SclReportItem> updateAllExtRefIedNames(Map<String, IEDAdapter> icdSystemVersionToIed) {
+        ExtRefService extRefService = new ExtRefService();
```

Comment:
You can use ExtRefService as instance variable instead of method variable 

Related to other PR #365 -- https://github.com/com-pas/compas-sct/pull/365/files#diff-b7f570f9aff5836fd28483972e8138d1e6d5b46d21fa730398c541d6256e3e6cR67

Reply:
OK